openlibrary/macros/FulltextSnippet.html

```diff
@@ -12,7 +12,7 @@
       $for snippet in snippets:
         $if snippet: 
           <li class="fulltext-excerpt">
-            &hellip;$:(snippet.replace("<", "&laquo;").replace(">", "&raquo;").replace("{{{", "<mark class='highlight'><strong>").replace("}}}", "</strong></mark>"))&hellip;
+            &hellip;$:(snippet.replace("<", "&laquo;").replace(">", "&raquo;").replace("{{{", "<mark class="highlight"><strong>").replace("}}}", "</strong></mark>"))&hellip;
```

Comment:
This doesn't seem related to HTML validation. It changes a single quote to a double quote, but either is valid.
Did you intend to add this?

Reply:
I read somewhere that " " for links are better than ' ' in HTML, that's why I change those otherwise there was no problem whatsoever